plugins/ibmcloud/api_key.go

```diff
@@ -0,0 +1,67 @@
+package ibmcloud
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIKey() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIKey,
+		DocsURL:       sdk.URL("https://www.ibm.com/docs/en/app-connect/container?topic=servers-creating-cloud-api-key"),
```

Comment:
> Use this information to create an IBM® Cloud API key that your deployed integration can use to authenticate

1Password Shell Plugins are built to be used by users, not by deployed integrations. Should we another type of authentication? Maybe username/password?

Reply:
I think this is the URL we're looking for:
```suggestion
		DocsURL:       sdk.URL("https://cloud.ibm.com/docs/account?topic=account-userapikey"),
```

Reply:
That looks right, thank you, Floris

Reply:
Updated!